server/bin/state/test-6.13.config/pbench-server.cfg

```diff
@@ -0,0 +1,54 @@
+install-dir = %(unittest-dir)s/opt/pbench-server
```

Comment:
What is the difference between the `test-6.13.config/pbench-server.cfg` file and the `test-6.14` one?  And further, can this be symlinks to existing files that are the same?